worlds/timespinner/Locations.py

```diff
@@ -217,7 +217,34 @@ def get_locations(world: Optional[MultiWorld], player: Optional[int]) -> Tuple[L
             LocationData('Left Side forest Caves', 'Cantoran',  1337176),
         )
 
-    # 1337177 - 1337236 Reserved for future use
+    # 1337177 - 1337198 Lore Checks
```

Comment:
Adding locations also required an update of data_version in _init_.py

Some locations do need thier requirements added, so a requirement is based on its region which has requirements to reach a certain region + additional requirement for certain checks like the breakable wall in the lab for the memories

Reply:
Added